### PR TITLE
Resource page timeline

### LINKED ISSUE #27: Add availability timeline to Resource page
- Change moments to strings.
- [x] Render simple timeline on resource page.
  - Hardcode date.
  - Simple onDateChange.
- [x] Allow changing date in timeline.
  - `actions/uiActions`
    - [x] Create `changeResourcePageTimelineDate(newDate)` action.
  - `state/reducers/resourcePage`
    - [x] On ENTER_OR_CHANGE_RESOURCE_PAGE, set date = location.query.begin || moment().format('...').
    - [x] On changeResourcePageTimelineDate, set date = action.payload.
  - `pages/resource/ResourcePageContainer`
    - [x] Take `date` as a prop.
    - [x] Define `onDateChange` as changeResourcePageTimelineDate.
  - `pages/resource/ResourcePage`
    - [x] Take `date` and give it to AvailabilityView.
  - `pages/resource/resourcePageSelector`
    - [x] Select date from state.

### PR DESCRIPTION
**Pull request by FelixLoether**
Closes #27.

### CODE REVIEW

**Comment by lraivio** [comment]
- [ ] I think we could make it so that on the resource page only the timeline without the sidebar is visible. The mvp version can be just to change the styles, unless it's super easy to make it on the Component level.

image

File: app/pages/resource/ResourcePageContainer.js [hunk 1]
```diff
@@ -1,37 +1,60 @@
...
+    this.handleDateChange = this.handleDateChange.bind(this);
+  }
+
   componentDidMount() {
     this.props.fetchResource(this.props.params.id);
   }
 
+  handleDateChange(date) {
+    const query = queryString.stringify({
+      ...this.props.location.query,
```
Comment (lraivio) [inline app/pages/resource/ResourcePageContainer.js]: I'm wondering what would be a good general way to handle cases where we want to only update one query paremeter in the url. Accessing `this.props.location.query` only works on Route level Components and adding the thing to propTypes is somewhat of a hassle. 
Something like `...browserHistory.getCurrentLocation().query` would work, but not sure if it has some downsides or is an un-reactish way to do things.

**Comment by FelixLoether** [comment]
Changes made.

**Comment by lraivio** [comment]
Nice job!

**Pull request merged**